library/src/main/java/com/google/android/exoplayer/text/eia608/Eia608TrackRenderer.java

```diff
@@ -158,14 +191,15 @@ protected boolean isReady() {
     return true;
   }
 
-  private void invokeRenderer(String text) {
+  private void invokeRenderer(SpannableStringBuilder text) {
     if (Util.areEqual(lastRenderedCaption, text)) {
       // No change.
       return;
     }
-    this.lastRenderedCaption = text;
+    final SpannableStringBuilder textToRender = text == null ? null : new SpannableStringBuilder(text);
```

Comment:
This line looks unnecessary. Have invokeRenderer accept a CharSequence and just pass it directly to the target?